src/apps/loan-list/materials/stackable-material/material-info.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, ReactNode } from "react";
```

Comment:
I should have told you earlier on. But we decided some time ago to use Pascal case in our component naming.
Which leaves us with a mixed naming setup, but that could be cleaned up in the future.
I know it is not optimal.